cirq-core/cirq/ops/gateset.py

```diff
@@ -268,7 +298,9 @@ def with_params(
             name: New name for the Gateset.
             unroll_circuit_op: If True, new Gateset will recursively validate
                 `cirq.CircuitOperation` by validating the underlying `cirq.Circuit`.
-            accept_global_phase_op: If True, new Gateset will accept `cirq.GlobalPhaseOperation`.
+            accept_global_phase_op: If True, new Gateset will include a `GateFamily` that accepts
+                `cirq.GlobalPhaseGate`. False or None have no effect. This parameter is deprecated
```

Comment:
Since `accept_global_phase_op=False` in the constructor of the class explicitly removes `cirq.GlobalPhaseGate` from the `*gates`, the same should be done here as well? So, the effect of values of `accept_global_phase_op` should be:
1. None - Have no effect
2. True - Always include cirq.GlobalPhaseGate to `*gates`. 
3. False - Always exclude cirq.GlobalPhaseGate from `*gates`.

Reply:
Yes, it does now, I just forgot to change the docstring.